packages/streamlib-webrtc/python/streamlib_webrtc/processors.py

```
@@ -369,42 +391,71 @@ def encoded_video(self) -> None:
+            except LinkOutputWriteFailure as write_failure:
+                # Not the endpoint's fault and not retryable: the bag this
+                # player wrote is one the engine refused, so reconnecting would
+                # spend an endpoint's session forever on a bag that will be
+                # refused again. Named, with its port, and the thread ends.
+                log.error(
+                    f"WhepPlayer: writing a bag on `{write_failure.port}` "
+                    f"failed, and the reading thread is ending: "
+                    f"{write_failure.__cause__}"
+                )
+                return
+            except Exception as failure:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)